spec/unit/harvestdor-indexer_spec.rb

```diff
@@ -153,17 +153,17 @@
     end
   end # whitelist
 
-  it "solr_client initializes the rsolr client using the options from the config" do
+  it 'solr_client initializes the rsolr client using the options from the config' do
     VCR.use_cassette('rsolr_client_config_call') do
-      indexer = Harvestdor::Indexer.new(Confstruct::Configuration.new(solr: { url: 'http://localhost:2345', a: 1 }))
+      indexer = described_class.new(Confstruct::Configuration.new(solr: { url: 'http://localhost:2345', a: 1 }))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.